src/Controller/ReportsController.php

```diff
@@ -112,16 +103,12 @@ public function getLatestReport(Course $course)
         return new JsonResponse($apiResponse);
     }
 
-    /**
-     * @Route("/download/courses/{course}/reports/pdf", methods={"GET"}, name="get_report_pdf")
-     * @param Course $course
-     * @return \Symfony\Component\HttpFoundation\Response
-     */
+    #[Route('/download/courses/{course}/reports/pdf', methods: ['GET'], name: 'get_report_pdf')]
     public function getPdfReport(
         Request $request,
         UtilityService $util,
         Course $course
-    ) {
+    ): \Symfony\Component\HttpFoundation\Response {
```

Comment:
I would suggest importing this at the top so you can just use `Response` here

Reply:
Yes, you're right, thank you for catching those. I believe those got jumbled while I was fixing merge conflicts.